pkg/appconsts/global_consts.go

```diff
@@ -30,7 +30,7 @@ const (
 	// DefaultUpgradeHeightDelay is the number of blocks after a quorum has been
 	// reached that the chain should upgrade to the new version. Assuming a block
 	// interval of 12 seconds, this is 7 days.
-	DefaultUpgradeHeightDelay = int64(7 * 24 * 60 * 60 / 12) // 7 days * 24 hours * 60 minutes * 60 seconds / 12 seconds per block = 50,400 blocks.
+	DefaultUpgradeHeightDelay = 3 // int64( 7 * 24 * 60 * 60 / 12) // 7 days * 24 hours * 60 minutes * 60 seconds / 12 seconds per block = 50,400 blocks.
```

Comment:
```suggestion
	DefaultUpgradeHeightDelay = 3 // int64( 7 * 24 * 60 * 60 / 6) // 7 days * 24 hours * 60 minutes * 60 seconds / 6 seconds per block = 100,800 blocks.
```

Reply:
[To reviewers] This change isn't intended for merging in this PR. It's a temporary workaround to address [this issue](https://github.com/celestiaorg/celestia-app/issues/3947#issuecomment-2401175659). I reduced the delay to ensure the e2e tests finish in a reasonable time—otherwise, we would have to wait for 50,400 blocks before the upgrade tests complete.